flax/linen/module.py

```diff
@@ -1488,12 +1488,61 @@ def init(self,
            **kwargs) -> FrozenVariableDict:
     """Initializes a module method with variables and returns modified variables.
 
+    ``init`` takes a ``PRNGKey`` as its first argument and all the ``*args`` and ``**kwargs``
```

Comment:
It takes either a single rng key or a dict mapping variable collections to rngs as the first argument, so maybe clarify that immediately? The current statement is incorrect.

Reply:
I originally did it for narrative purposes, you are right, fixed.